packages/shared/src/components/auth/AuthOptions.tsx

```diff
@@ -161,6 +165,7 @@ function AuthOptions({
   const { displayToast } = useToastNotification();
   const { syncSettings } = useSettingsContext();
   const { logEvent } = useLogContext();
+  const onboardingPapercuts = useFeature(featureOnboardingPapercuts);
```

Comment:
You need specific enrolment?
I think authOptions also get's used outside onboarding page right?

Reply:
Ah ok, so you don't care about enrollment only from onboarding/new users?
Just checking as I'm not sure what you guys decided on.

Reply:
But yeah, basically, wherever the components are rendered, we should enroll.

Reply:
Ah ok if it's from GB side it's fine :) 
Just making sure everyone's aligned as enrollment is almost instant